test/test.js

```diff
@@ -22,12 +35,38 @@ test('equals to sample', async t => {
   fs.writeFileSync(dest, zip, 'binary');
 
   t.ok(zip instanceof Buffer);
+});
+
+test('check internal structure', async t => {
+  // Create deck as in previous example
+  const apkg = new AnkiExport('deck-name');
+  const cards = [
+    { front: 'card #1 front', back: 'card #1 back' },
+    { front: 'card #2 front', back: 'card #2 back' },
+    { front: 'card #3 with image <img src="anki.png" />', back: 'card #3 back' }
+  ];
+  addCards(apkg, cards);
+  const zip = await apkg.save();
+  fs.writeFileSync(dest, zip, 'binary');
+
+  // extract dec to tmp directory
+  await decompress(dest, destUnpacked);
+  // analize db via sqlite
+  const db = new sqlite3.Database(destUnpackedDb);
+  const result = await new Promise((resolve, reject) => db.all(
+    `SELECT
+      notes.sfld as front,
+      notes.flds as back
+      from cards JOIN notes where cards.nid = notes.id ORDER BY cards.id`,
+    (err, data) => err ? reject(err) : resolve(data)
+  ));
+  db.close();
 
-  const sampleZip = fs.readFileSync(sample);
-  const destZip = fs.readFileSync(dest);
+  // compare content from just created db with original list of cards
+  const normilizedResult = _.sortBy(result.map(({front, back}) => ({
+    front,
+    back: back.split(SEPARATOR).pop()
+  })), 'front');
 
-  // t.ok(destZip === sampleZip);
-  //
-  // i forgot that there are random number involved, so every time slightly different result
-  // one of solutions could be comparing sql data extracted from the apkg archive
+  t.ok(_.isEqual(normilizedResult, cards));
```

Comment:
Please change to `t.deepEqual(normalizedResult, card)`. `ava` uses `lodash#isEqual` internally https://github.com/avajs/ava/blob/82677554efe223fddcfa9f89db69a106611d257e/lib/assert.js#L4